app/routes/edit-account.js

```diff
@@ -0,0 +1,7 @@
+import AuthenticatedRoute from 'ember-authenticate-me/routes/authenticated';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'AuthenticatedRoute' is not defined.<br>'from' is not defined.